tests/datasource/fluent/conftest.py

```diff
@@ -483,3 +486,24 @@ def _source(
 @pytest.fixture
 def create_source() -> ContextManager:
     return _source  # type: ignore[return-value]
+
+
+@pytest.fixture
+def sql_datasource_table_asset_test_connection_noop(
```

Comment:
This fixture was moved from `test_sql_datasources.py` to `conftest.py` so I could reuse it in `test_sql_asset.py`. Both these test files test functionality in `sql_datasource.py`.